src/main.rs

```diff
@@ -83,7 +81,7 @@ async fn handle_keypress(
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     let mut hdr_buf = [0u8; MSG_HDR_SIZE];
-    let mut buf = [0u8; SSH_BUFFER_SIZE];
+    let mut buf: [u8; 2048] = [0u8; SSH_BUFFER_SIZE];
```

Comment:
??

Reply:
oops looks like in my moving around, I got git confused when fixing up merge conflicts that were created by fixups..